typings/tests.ts

```diff
@@ -447,6 +446,66 @@ client.on('ready', async () => {
 // This is to check that stuff is the right type
 declare const assertIsPromiseMember: (m: Promise<GuildMember>) => void;
 
+const baseCommandOptionData = {
+  name: 'test',
+  description: 'test',
+};
+
+assertType<ApplicationCommandOptionData>({
+  ...baseCommandOptionData,
+  type: 'STRING',
+  // @ts-ignore
```

Comment:
Do we not want this instead?
```suggestion
  // @ts-expect-error
```